src/examples/rest/src/backend/schema/task.ts

```diff
@@ -93,7 +93,7 @@ export class Task {
 	@Field(() => Boolean)
 	isCompleted!: boolean;
 
-	@RelationshipField<OrmTask>(() => User, { id: 'userId' })
+	@RelationshipField<OrmTask>(() => User, { id: (entity) => entity.userId })
```

Comment:
Interesting, this should work with the `'userId'`, no? If not, we probably want to fix that up.

Reply:
Cool I was not sure if that was by design, It currently fails with the error:

`Cannot return null for non-nullable field Task.user.`

Reply:
Right, let's go ahead and merge this and create a backlog task to restore the string behaviour for simple field things.

Reply:
it is restored now as part of this PR